src/xrEngine/profiler.cpp

```diff
@@ -14,9 +14,9 @@
 #include "xrEngine/GameFont.h"
 
 #ifdef CONFIG_PROFILE_LOCKS
-static volatile LONG					critical_section_counter = 0;
+static volatile signed int critical_section_counter = 0;
```

Comment:
For now, I don't know why it was been made `LONG`. Let's leave it as is for now.
```suggestion
static volatile LONG					critical_section_counter = 0;
```